media/js/src/app.jsx

```diff
@@ -1,8 +1,9 @@
 import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router, Route, Routes  } from 'react-router-dom';
-import { Dashboard } from './dashboard';
-import { SimulationOne } from './simulationOne/simulationOne';
-import { SimulationTwo } from './simulationTwo/simulationTwo';
+import { Dashboard } from './containers/dashboard';
+import { SimulationOne} from './simulations/simulation1/simulationOne';
+// eslint-disable-next-line max-len
+import { SimulationTwo } from './simulations/simulation2/components/simulationTwo';
```

Comment:
Instead of disabling eslint I usually format imports like this, which also works from multiple imports:
```
import {
    SimulationTwo
} from './simulations/simulation2/components/simulationTwo';
```